Formula/lesspipe.rb

```diff
@@ -16,7 +17,12 @@ class Lesspipe < Formula
   end
 
   def install
-    system "./configure", "--prefix=#{prefix}"
+    args = %W[
+      --prefix=#{prefix}
+    ]
+    # Newer shell features not present in macOS bash
```

Comment:
should we just `depends_on bash` for the new shell features?

Reply:
This alone would be insufficient. Their configure script [strips out](https://github.com/wofr06/lesspipe/blob/a24c496dd1d820795be8ddc624d65d8504fb5dab/configure#L122) the dirname of the shell and uses only `#!/usr/bin/env $selected_shell` so homebrew bash would also need to be earlier in the PATH.
Some options I can think of are:
* Alter the configure script upstream to not do this
* Substitute this line with a direct link to homebrew's bash
* Submit a PR upstream to not use the instance of `typeset -l`
* Use pre-installed zsh

My preference was for switching to zsh, as this doesn't install a new dependency just for a script

Reply:
> Submit a PR upstream to not use the instance of typeset -l

can we log an upstream issue for that, I feel like that is a better approach, no?

Reply:
Opened an issue for it here: https://github.com/wofr06/lesspipe/issues/121